package.json

```diff
@@ -76,8 +76,8 @@
     "eslint-plugin-standard": "^4.1.0",
     "mjs-entry": "gist:de6257751f54b3c66319bae8d2a8aea0",
     "prettier": "2.2.1",
-    "typedoc": "^0.20.13",
-    "typescript": "^4.1.3"
+    "typedoc": "^0.20.29",
+    "typescript": "~4.2.3"
```

Comment:
Any rationale behind `~` not `^`?

Reply:
To avoid breaking the build for users of alternative package managers (minor TypeScript releases contain breaking changes)